examples/lighting-app/esp32/README.md

```diff
@@ -128,6 +128,25 @@ is "nvs".
 `scripts/tools/generate_esp32_chip_factory_bin.py` script generates the
 chip-factory NVS image `partition.bin`.
 
+Script has dependency on
```

Comment:
LGTM. Generic comment: Should we move the entire `Using ESP32 Factory Data Provider` section before `-   To build the demo application.`?

Reply:
Moved to "Building the Example Application"